docs/dev/02-making-changes.md

```diff
@@ -14,11 +14,21 @@ Here are some tips on how to set up a Karma workspace and how to send a good pul
 
 * Make sure you have a [GitHub account](https://github.com/signup/free).
 * [Fork the repository] on GitHub.
-* Clone your fork.
+* Clone your fork
   ```bash
   $ git clone https://github.com/<your-username>/karma.git
   $ cd karma
   ```
+* Install for development. Use a recent npm version, ignore peerdep warnings
+  ```bash
+  $ rm -rf node_modules
```

Comment:
`--dev` is not needed, by default all dependencies are installed including dev dependencies